__tests__/app.test.js

```diff
@@ -496,4 +496,37 @@ describe("GET /api/reviews/:review_id (comment count)",() => {
             expect(review.review_id).toEqual(2)
         })
     })
+})
+
+// DELETE /api/comments/:comment_id
+describe("DELETE /api/comments/:comment_id",() => {
+    test("if given valid comment_id, status 204, return nothing - check that number of comments is reduced",() => {
+        return request(app).get('/api/reviews/2/comments')
+        .expect(200)
+        .then(({body:{comments}}) => {
+            expect(comments).toHaveLength(3)
+            return request(app).delete('/api/comments/5')
+            .expect(204)
```

Comment:
Great to see the 204 here, its easy after dealing with lots of get requests to default to 200